rero_ils/modules/templates/extensions.py

```diff
@@ -52,37 +44,5 @@ def post_init(self, record, data, model=None, **kwargs):
             else:
                 record.get('data', {}).pop(field, None)
 
-
-class TemplateVisibilityChangesExtension(RecordExtension):
-    """Disable template visibility changes depending on connected user."""
-
-    def pre_commit(self, record):
-        """Called before a record is committed.
-
-        :param record: the record containing data to validate.
-        :raises ValidationError: If an error is detected during the validation
-            check. This error could be serialized to get the error message.
-        """
-        # First, determine if a user is connected. If not, no check must be
-        # done about any changes (probably it's a console script/user).
-        from rero_ils.modules.patrons.api import current_librarian
-        if not current_user:
-            return
-
-        # Check if visibility of the template changed. If not, we can stop
-        # the validation process.
-        original_record = record.db_record() or {}
-        if record.get('visibility') == original_record.get('visibility'):
-            return
-
-        # Only lib_admin and full_permission roles can change visibility field
-        error_message = "You are not allowed to change template visibility"
-        allowed_roles = [
-            UserRole.FULL_PERMISSIONS,
-            UserRole.LIBRARY_ADMINISTRATOR
-        ]
-        user_roles = set()
-        if current_librarian:
-            user_roles = set(current_librarian.get('roles'))
-        if not user_roles.intersection(allowed_roles):
-            raise ValidationError(error_message)
+    pre_create = _clean_record
```

Comment:
Be careful the `pre_create` modify only the dict data not the db data. You should probably do something like: https://github.com/rero/rero-ils/blob/683067bf05f62c3e1f17c6424509dbd3a13b2e69/rero_ils/modules/record_extensions.py#L59